isis/src/mro/apps/hideal2pds/tsts/notJitterCorrected/Makefile

```diff
@@ -19,7 +19,7 @@ include $(ISISROOT)/make/isismake.tsts
 
 commands:
 	$(APPNAME) FROM=$(INPUT)/input.cub \
-	  to=$(OUTPUT)/output.img > /dev/null;
+	  to=$(OUTPUT)/output.img bits=16 > /dev/null;
```

Comment:
Do we need to add a note about backwards compatibility because this now outputs 10 bit by default instead of 16 bit?

Reply:
I cannot find this issue, so it may be a matter of me not understanding context, but why IS the default 10 bit?

Reply:
It looks like HiRISE requested the default be changed. It's back on the Redmine ticket for this work.

Reply:
Yes, Tyler's change was at the request of HiRISE.

Reply:
updated